testcafe/searchField.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Console } from "console";
+import { Selector } from "testcafe";
+import { Config } from "./testcafe.config";
+
+fixture`Search field test`.page`${Config.baseUrl}`
+    .beforeEach( async t => {
+        const nextButton = Selector("a").withAttribute("role", "button").child().withText("Weiter");
+        const websiteText = Selector("html").textContent;
```

Comment:
Getting the whole text of the website has shown to be unreliable. In `introScreens.spec.ts` on master this functionality is slightly refactored. It might be good to use it in here too.

It might even make sense to create a common "page object" to have the logic in one place. Something like "IntroPage.ts" with a function: `finishIntroScreensWith(plz?: string)`.

Reply:
Getting the whole text is not reliable, hidden text are always appearing in the Selector("html").textContent String.

Reply:
I'm leave the assertion as it is, any held appreciate.

Reply:
I added a PR that changes this branch with some changes to get rid of code duplication and to not use the whole html text content.
https://github.com/CovOpen/CovMapper/pull/349